webapp/src/userSettings.ts

```diff
@@ -52,6 +52,14 @@ export class UserSettings {
         UserSettings.set(UserSettingKey.WelcomePageViewed, newValue)
     }
 
+    static get dashboardShowEmpty(): boolean {
+        return localStorage.getItem('dashboardShowEmpty') !== 'false'
+    }
+
+    static set dashboardShowEmpty(newValue: boolean) {
+        localStorage.setItem('dashboardShowEmpty', JSON.stringify(newValue))
+    }
+
```

Comment:
Can we add the new item name to the `UserSettingKey` enum above? Also call the local methods for consistency.

Reply:
Thanks, done. I originally made the changes on the release-0.9.0 branch, and missed this enum.